### ISSUE TITLE
A11y_Azure Data Studio_Query Plan (Preview)_Keyboard: Tooltip provided at each not is not accessible with keyboard.

### ISSUE THREAD

**Issue by poojapatil36** [microsoft/azuredatastudio#20749]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #Win11 #Keyboard; #WCAG2.1.1; #FTP;

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:

1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Observe that tooltip provided at each not is not accessible with keyboard.

### Actual:
Tooltip provided at each not is not accessible with keyboard.

**NOTE**: The issue is reproducing throughout the application wherever tooltip is present. 

### Expected Result:
Tooltip provided at each node should be accessible with keyboard

### Attachment:
![A11y_Azure Data Studio_Query Plan (Preview)_Keyboard](https://user-images.githubusercontent.com/97668775/194281255-c43d669a-7bec-4b99-8c35-f2cca3c80632.gif)

**Comment by alanrenmsft** [comment]
@poojapatil36  
dupe of: https://github.com/microsoft/azuredatastudio/issues/19566
we've already have a sev3 issue for AzureDataStudio.
please close the issue.

**Comment by poojapatil36** [comment]
Issue is fixed in latest insider build. Please find below attachment for reference.
![Presentation1](https://user-images.githubusercontent.com/97668775/216571918-6d1cba12-30c6-4430-b04f-86bb9aa04111.gif)

**Comment by 12-shweta** [comment]
Github Tags: #Closed;
